mhs/inbound/nhs-inbound-env-example.yaml

```diff
@@ -1,12 +1,12 @@
 #Common env variables for inbound
-MHS_INBOUND_HEALTHCHECK_SERVER_PORT: "8081"
+MHS_INBOUND_HEALTHCHECK_SERVER_PORT: "8082"
```

Comment:
Wound up changing it to 8082 because I tested outbound (8081) first. Can be reverted if we need to, but the postman collection matched these ports now.